app/views/static/blog/_posts/2015-01-20-give-our-sorting-office-a-try.md

```diff
@@ -0,0 +1,35 @@
+---
+author: Stuart Harrison
+title: Give our sorting office a try!
+---
+
+Addresses are hard. Really hard. Especially if you're a programmer. We've been applying addresses to buildings since before the explosion in computing technology, and it shows.
+
+Addresses are full of strange exceptions and things that programmers might miss when taking user input and trying to apply machine-like logic to it. (There's a [blog post called 'Falsehoods programmers believe about addresses'](https://www.mjt.me.uk/posts/falsehoods-programmers-believe-about-addresses/) which highlights some of the most well-known problems, if you want to read all the gory details.)
+
+As the name of our particular game is addresses, this has been an issue we've had to get to grips with at an early stage, and, as we start accepting submissions from other developers, we know it's an issue that they'll have to deal with too.
+
+For part of the past two weeks, we've tried to try and ease some of that pain for those who would come after us, and have built [an application that tries to take a single address and break it down into its constituent parts](https://github.com/OpenAddressesUK/sorting_office).
```

Comment:
@pezholio I've discussed this with Jeni and this could be a good opportunity to credit Fusion Data Science for the experience and the algorithms they contributed.